decentraland/unity-renderer
# refactor: avatar lods using new feature flag system

## What does this PR change?

This PR migrate the avatar lods feature flag to the new feature flag system so we can remove the legacy code from kernel

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=refactor/avatar-lod-flag
2. Check that the avatars lods is working 

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
